scripts/core/editor3/reducers/editor3.jsx

```diff
@@ -55,20 +55,32 @@ export const forceUpdate = (state) => {
 /**
  * @ngdoc method
  * @name onChange
+ * @param {Object} state
  * @param {Object} editorState
+ * @param {Bool} update if true force change value and editor refresh
  * @return {Object} returns new state
  * @description Handle the editor state has been changed event
  */
-export const onChange = (state, newState) => {
+export const onChange = (state, newState, update = false) => {
```

Comment:
Please add a line here:
```js
// TODO(x): Remove `force` once Draft v0.11.0 is in
```
I have other TODOs like this in editor3, making it easier for us to update later on.